src/bblgum/impl.clj

```diff
@@ -30,3 +30,107 @@
                        str/trim
                        str/split-lines
                        (filter seq))))))
+
+(defn gum*
```

Comment:
lets call this exec, simplify the docstring.

Reply:
we already have exec :)

Reply:
ah yes, `gum*` is fine then!